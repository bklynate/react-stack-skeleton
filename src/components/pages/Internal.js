/* @flow */
import React from 'react';

type Props = {

};

/**
 * The home page of the website.
 */
export default function Internal({ ...otherProps }: Props): React.Element<*> {
  return (
    <div {...otherProps}>
      Internal Page
    </div>
  );
}
