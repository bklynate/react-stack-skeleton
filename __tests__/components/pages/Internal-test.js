import React from 'react';
import renderer from 'react-test-renderer';

import Internal from '../../../src/components/pages/Internal';

test('The Internal component displays another internal page.', () => {
  const component = renderer.create(
    <Internal />
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
