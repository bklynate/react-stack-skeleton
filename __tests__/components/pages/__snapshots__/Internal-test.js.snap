exports[`test The Internal component displays another internal page. 1`] = `
<div>
  Internal Page
</div>
`;
